### ISSUE TITLE
Lots of warnings during compilation with recent gcc-C++ and Qt

### ISSUE THREAD

**Issue by monnerat** [chris2511/xca#149]
Fortunately these are only warnings. But they are a lot of the same type.
Here is an example:

```
In file included from /usr/include/qt5/QtCore/QVariant:1,
                 from /builddir/build/BUILD/xca-2.2.0/ui/ui_KeyDetail.h:12,
                 from KeyDetail.h:11,
                 from moc_KeyDetail.cpp:9:
/usr/include/qt5/QtCore/qvariant.h: In constructor 'QVariant::QVariant(QVariant&&)':
/usr/include/qt5/QtCore/qvariant.h:273:25: warning: implicitly-declared 'constexpr QVariant::Private& QVariant::Private::operator=(const QVariant::Private&)' is deprecated [-Wdeprecated-copy]
  273 |     { other.d = Private(); }
      |                         ^
/usr/include/qt5/QtCore/qvariant.h:399:16: note: because 'QVariant::Private' has user-provided 'QVariant::Private::Private(const QVariant::Private&)'
  399 |         inline Private(const Private &other) Q_DECL_NOTHROW
      |                ^~~~~~~
```
Using gcc-c++ 9.2.1
Using Qt 5.12.5

I don't know if the problem can be solved in XCA or lies in Qt.

**Comment by chris2511** [comment]
Seems to be a QT + gcc-9 issue: https://codereview.qt-project.org/c/qt/qtbase/+/245434
Will be fixed in QT 5.13

**Comment by monnerat** [comment]
Thanks for your investigations.
Closing issue, as it should be fixed externally.
